src/main/java/org/sagebionetworks/web/client/context/KeyFactoryProviderImpl.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.web.client.context;
+
+import org.sagebionetworks.web.client.jsinterop.KeyFactory;
+
+public class KeyFactoryProviderImpl implements KeyFactoryProvider {
+
+  @Override
+  public KeyFactory getKeyFactory(String currentUserAccessToken) {
+    return new KeyFactory(currentUserAccessToken);
+  }
+}
```

Comment:
Create a mockable/injectable KeyFactoryProvider so I can easily mock KeyFactory in tests that don't support JsInterop